src/js/components/Cards/stories/Children.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
I think the Cards stories could use some extra spacing. They look fine in the hpe theme but in the grommet theme the text is right against the edge of the card.

<img width="812" alt="Screen Shot 2022-12-08 at 3 07 58 PM" src="https://user-images.githubusercontent.com/54560994/206577641-1b2228a8-6bc6-468c-93d3-09c4eb5697d1.png">

Reply:
Hmm... I'm wondering if we should address this in the base or grommet themes?

Reply:
It does feel like the card content should have some default padding, but not sure how we can add that into the theme without breaking backwards compatibility

Reply:
I think we could consider adding it to the `grommet` theme and consider the current one a bit broken. We wouldn't break anyone's custom theme derived from `base`.

Reply:
I'm good with adding it to the grommet theme

Reply:
since we have `cards.container` in theme can we do `cards.container.card.pad` or would that be weird